modules/raxml/raxml.nf

```diff
@@ -0,0 +1,27 @@
+nextflow.enable.dsl = 2
+
+params.results_dir = "${params.outdir}/raxml"
+params.save_mode = 'copy'
+params.should_publish = true
+
+process RAXML {
+    publishDir params.results_dir, mode: params.save_mode, enabled: params.should_publish
+
+    input:
+    path(alignment_file)
+    output:
+    path("*")
```

Comment:
@Mxrcon , I'm not sure if this is a good idea, this would end up publishing `.command.xyz` files as well I think.

The standard practice is to specify exactly, what files are needed.

Reply:
yes, I need to work this a little bit, I'm not to used on raxml usage